app/utils/ipfs.js

```diff
@@ -47,17 +52,47 @@ export async function saveText(text) {
 }
 
 async function saveFileTheGraph(buf) {
-  const saveResult = await getIpfsApiTheGraph().add(buf);
-  // return saveResult.cid.toString();
+  return await getIpfsApiTheGraph().add(buf);
+}
+
+function prepareFile(data) {
```

Comment:
One line function that is used only once does not give any value

Reply:
done